apps/admin-server/src/pages/projects/[project]/crypto.tsx

```diff
@@ -0,0 +1,96 @@
+import * as crypto from "crypto";
```

Comment:
Misschien zie ik iets over het hoofd, maar gebruik je dit wel? Is dat inmiddels niet naar de API verhuisd?